.github/workflows/mypy.yaml

```diff
@@ -19,7 +19,7 @@ jobs:
           python-version: 3.8
 
       - name: Install
-        run: pip install mypy pydantic
+        run: pip install "mypy==0.910" pydantic
```

Comment:
@timj does this want a comment, like "pin to address compatibility issue between mypy 0.92 and pedantic"?